app/components/pages/Dashboard/Dashboard.js

```diff
@@ -1,90 +1,78 @@
 import React from 'react'
 import styled from 'styled-components'
-import { Link } from 'react-router-dom'
-import { Box } from 'grid-styled'
+import { Box, Flex } from 'grid-styled'
 import { Button } from '@pubsweet/ui'
-
+import { th } from '@pubsweet/ui-toolkit'
 import media from '../../global/layout/media'
-import { FormH2, FormH3 } from '../../ui/atoms/FormHeadings'
-import Paragraph from '../../ui/atoms/Paragraph'
-import ManuscriptStatus from '../../ui/atoms/ManuscriptStatus'
+import SmallParagraph from '../../ui/atoms/SmallParagraph'
+import ExternalLink from '../../ui/atoms/ExternalLink'
 import StickyFooter from '../../ui/atoms/StickyFooter'
+import Tabs from '../../ui/molecules/Tabs'
+import DashboardList from '../../ui/molecules/DashboardList'
+import { ALL_MANUSCRIPTS } from './oporations'
+import ManuscriptsQuery from './ManuscriptsQuery'
 
-/* Temporary dashboard view pending receipt of designs */
-
-const BoxHiddenFromMobile = styled(Box)`
+const RightAlignedButton = styled(Box)`
```

Comment:
This naming kinda makes it sound like styling has been applied to an actual `<Button>`, whereas what you are renaming is a `Box` (a `div` under the hood).

In this specific situation, your div's content happens to be a button :+1: But in another situation, you might put something completely different inside. This Box doesn't care what it's contents are. So long as you want to right-align the contents & hide it from mobile, you want to use this same Box. So I reckon the naming needs to reflect those 2 aspects of what this Box does, rather than what goes inside it. Does that make sense?

So what do you choose that's going to encapsulate those 2 pieces of functionality about this Box, i.e. those 2 additional bits of CSS. Well...`RightAlignedBoxHiddenFromMobile` is really long. And the minute you add another bit of CSS that makes it blue, it becomes `BlueRightAlignedBoxHiddenFromMobile`. And so on & so on & so on.

So, whilst I understand [the desire for more declarative naming](https://github.com/elifesciences/elife-xpub/pull/805#discussion_r225962686), I think that, in general, this is why you often see `StyledBox` or `StyledHeader`, `StyledParagraph`, etc - it's more extensible.

`StyledXYZ` usually seems the most appropriate to me, until it's such a common pattern throughout your application that it needs to be its own `atom`.

Feel free to disagree, of course.

Reply:
This makes sense, I'll change this back to `BoxHiddenFromMobile` for the time being then.